app/models/twilio_notifier.rb

```diff
@@ -0,0 +1,29 @@
+require 'twilio-ruby'
+
+class TwilioNotifier
+  def initialize(order)
+    @order = order
+    
+  end
+
+  def ordered_item_titles
+    @order.items.pluck(:title).to_sentence
+  end
+
+  def notify
+    client.messages.create(
+      from: '+19495372518',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.